src/styles/style.scss

```diff
@@ -0,0 +1,140 @@
+.page {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  color: #060b35;
+  width: 200px;
+
+  height: 408px;
+  display: flex;
+  flex-direction: column;
+}
+
+.page__body {
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+}
+
+.image {
+  display: flex;
+}
+
+.image_mac {
+  padding: 32px 19px 40px;
+  height: 134px;
+  width: 160px;
+  align-items: center;
+}
+
+.header__text {
+  padding: 0 16px;
+}
+
+.header__title {
+  margin: 0;
+  font-weight: 500;
+  line-height: 18px;
+  font-size: 12px;
+  white-space: nowrap;
+}
+
+.header__product {
+  margin: 0;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+  color: #616070;
+  padding: 4px 0 16px;
+}
+
+.review {
+  display: flex;
+}
+
+.stars {
+  display: flex;
+  padding-left: 16px;
+}
+
+.stars-wrapper {
+  padding-bottom: 24px;
+  display: flex;
+  align-items: center;
+}
+
+.review__text {
+  margin: 0;
+  font-weight: 400;
+  line-height: 14px;
+  font-size: 10px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-image: url(../images/star.svg);
+}
+
+.stars__star.is-active {
+  background-image: url(../images/star.svg);
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
Move stars styles to the separate file

Reply:
not fixed

Reply:
<img width="690" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/908818e9-6d8d-4351-8e20-87437df3c38e">